<?xml version="1.0"?>
<launch>
  <arg name="ns"            default="camera"/>
  <arg name="image"         default="color/image_raw"/>
  <arg name="camera_info"   default="color/camera_info"/>
  <arg name="marker_size"   default="0.0315"/>

  <group ns="$(arg ns)">
      <node name="detector_node" pkg="aruco_detector" type="detector_node" output="screen">
        <param name="image"         value="$(arg image)" />
        <param name="camera_info"   value="$(arg camera_info)" />
        <param name="marker_size"   value="$(arg marker_size)" />
        <param name="tf_prefix"     value="$(arg ns)" />
      </node>
   </group>
</launch>